web/Controllers/AdminController.cs

```diff
@@ -50,4 +50,25 @@ public IActionResult RequestAdminAccess()
 
         return RedirectToAction(nameof(Index));
     }
+
+    public IActionResult CancelAdminAccessRequest()
+    {
+        var user = GetCurrentUser();
+        if (user.IsAdmin)
+        {
+            _logger.LogWarning("User {UserId} already an admin",user.Id);
+            TempData["FlashType"] = "danger";
+            TempData["FlashMessage"] = "Already an admin";

```

Comment:
We should pull this out into a method on AbstractController at some point since we're using it quite a bit. Definitely separate PR though

Reply:
Will make a ticket :)